pkgs/top-level/python-aliases.nix

```diff
@@ -142,6 +143,7 @@ mapAliases ({
   doctest-ignore-unicode = throw "doctest-ignore-unicode has been removed since it has been unmaintained for 11 years"; # added 2024-05-20
   dogpile_cache = dogpile-cache; # added 2021-10-28
   dogpile-core = throw "dogpile-core is no longer maintained, use dogpile-cache instead"; # added 2021-11-20
+  dyn = throw "dyn has been removed because it is no longer maintained"; # added 2024-05-20
```

Comment:
I'm not familiar with this package, but it doesn't seem to be unmaintained since the last release was a year ago.

Reply:
Thanks for your work!